cmd/root_test.go

```diff
@@ -1348,6 +1348,41 @@ func TestQuitQuitQuit(t *testing.T) {
 	}
 }
 
+func TestQuitQuitQuitHttpGet(t *testing.T) {
+	c := NewCommand(WithDialer(&spyDialer{}))
+	c.SilenceUsage = true
+	c.SilenceErrors = true
+	c.SetArgs([]string{"--quitquitquit", "--admin-port", "9192", "my-project:my-region:my-instance"})
```

Comment:
Let's use 9193 or some other port to avoid conflicts with the test above.

Reply:
Fixed.